src/modlunky2/ui/levels.py

```diff
@@ -31,13 +33,52 @@
 from modlunky2.levels.monster_chances import MonsterChance, MonsterChances
 from modlunky2.levels.tile_codes import VALID_TILE_CODES, TileCode, TileCodes
 from modlunky2.sprites import SpelunkySpriteFetcher
-from modlunky2.sprites.tilecode_extras import TILENAMES
 from modlunky2.ui.widgets import PopupWindow, ScrollableFrameLegacy, Tab
 from modlunky2.utils import is_windows, tb_info
 
 logger = logging.getLogger("modlunky2")
 
 
+class EditorType(Enum):
+    VANILLA_ROOMS = "single_room"
+    CUSTOM_LEVELS = "custom_levels"
+
+
+class CustomLevelSaveFormat:
+    def __init__(self, name, room_template_format, include_vanilla_setrooms):
+        self.name = name or room_template_format
+        self.room_template_format = room_template_format
+        self.include_vanilla_setrooms = include_vanilla_setrooms
+
+    @classmethod
+    def LevelSequence(cls):
```

Comment:
python functions and methods use `snake_case` not `CamelCase`

Reply:
Done.